api_collections/booking_api.py

```diff
@@ -0,0 +1,33 @@
+from utilities.api_utilities.base_api import BaseAPI
+
+
+class BookingAPI(BaseAPI):
+
+    def __init__(self, environment):
+        super().__init__(environment)
+        self.__booking_url = '/booking'
+
+    def setup_token(self, environment):
+        from api_collections.auth_api import AuthAPI
```

Comment:
can you import it globally?

Reply:
initially there was identical naming for method, so to avoid circular import, it was done locally.

Reply:
moved out